tools/test-setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euxo pipefail
```

Comment:
I recently learned that adding `-E` will make it even safer:
```suggestion
set -Eeuxo pipefail
```